command/cp.go

```diff
@@ -302,7 +309,8 @@ func (c Copy) Run(ctx context.Context) error {
 		}
 	}()
 
-	isBatch := srcurl.HasGlob()
+	// use --raw flag to prevent glob operations.
```

Comment:
you can remove this comment since we don't check --raw here anymore.

Reply:
deleted comment line.